adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenInteractiveHandler.cs

```diff
@@ -205,6 +205,11 @@ private DictionaryRequestParameters CreateAuthorizationRequest(string loginHint)
             authorizationRequestParameters[OAuthParameter.HasChrome] = "1";
             authorizationRequestParameters[OAuthParameter.RedirectUri] = this.redirectUriRequestParameter;
 
+#if DESKTOP
+            // Added form_post as a way to request to ensure we can handle large requests for dsts scenarios
+            authorizationRequestParameters[OAuthParameter.ResponseMode] = OAuthResponseModeType.FormPost;
```

Comment:
>authorizationRequestParameters[OAuthParameter.ResponseMode] = OAuthResponseModeType.FormPost; [](start = 12, length = 93)

this one will require extensive testing. Also we should ensure to kick of an automation run on this PR before committing.

Reply:
Agreed, this would require extensive testing as this would be hit in existing code paths. Evo will not ignore this and will handle it in the same way.